crates/ruma-client-api/src/room/get_summary.rs

```diff
@@ -0,0 +1,100 @@
+//! `GET /_matrix/client/v1/summary/{roomIdOrAlias}`
+//!
+//! Experimental API enabled with MSC3266.
+//!
+//! Returns a short description of the state of a room, with state events.
+
+pub mod msc3266 {
+    //! `MSC3266` ([MSC])
+    //!
+    //! [MSC]: https://github.com/deepbluev7/matrix-doc/blob/room-summaries/proposals/3266-room-summary.md
+
+    use ruma_common::{
+        api::{request, response, Metadata},
+        metadata,
+        room::RoomType,
+        space::SpaceRoomJoinRule,
+        EventEncryptionAlgorithm, OwnedMxcUri, OwnedRoomAliasId, OwnedRoomId, OwnedRoomOrAliasId,
+        OwnedServerName, RoomVersionId,
+    };
+    use ruma_events::room::member::MembershipState;
+
+    const METADATA: Metadata = metadata! {
+        method: GET,
+        rate_limited: false,
+        authentication: AccessTokenOptional,
+        history: {
+            unstable => "/_matrix/client/unstable/im.nheko.summary/rooms/:room_id_or_alias/summary",
+        }
+    };
+
+    /// Request type for the `summary/room_id_or_alias` endpoint.
```

Comment:
I believe we use the name of the module here in general:

```suggestion
    /// Request type for the `get_summary` endpoint.
```